### ISSUE TITLE
Port twisted.web._stan to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7952]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7952|
|Type|enhancement|
|Created|2015-06-26 06:56:12Z|
|Branch|https://github.com/twisted/twisted/tree/twstan-py3-7952|

This is used by ```twisted.web.template```, and we should port it.

<details><summary>Searchable metadata</summary>

```
trac-id__7952 7952
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_twstan_py3_7952 branches/twstan-py3-7952
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1435301772738022 1435301772738022
changetime__1435494404011833 1435494404011833
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45110]) Branching to twstan-py3-7952.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

This is looking in pretty good shape. This'll be a stepping stone to porting ```twisted.web.template``` which should clean up the rest of the coverage.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

All good. Please merge. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45119]) Merge twstan-py3-7952: Port twisted.web._stan to Python 3

Author: hawkowl
Reviewer: adiroiban
Fixes: [#7952](https://github.com/twisted/twisted/issues/7952)
